include/ros1_bridge/factory.hpp

```diff
@@ -45,7 +45,7 @@ class Factory : public FactoryInterface
     const std::string & topic_name,
     size_t queue_size)
   {
-    rmw_qos_profile_t custom_qos_profile = rmw_qos_profile_sensor_data;
+    rmw_qos_profile_t custom_qos_profile = rmw_qos_profile_default;
```

Comment:
I didn't do it here because it's used in the next line but I did for `simple_bridge_1_to_2` in https://github.com/ros2/ros1_bridge/pull/37/commits/08881e49b207237b8372539fddaa84df9e77f8e6